rclpy/test/test_type_hash.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from rclpy.type_hash import TypeHash
+
+
+STD_MSGS_STRING_TYPE_HASH_DICT = {
+    'version': 1,
+    'value': b'\xdf\x66\x8c\x74\x04\x82\xbb\xd4\x8f\xb3\x9d\x76\xa7\x0d\xfd\x4b\xd5\x9d\xb1\x28'
```

Comment:
tiny nit: formatting is a little odd with line break 20+12, might as well go 16+16.